### ISSUE TITLE
Ever thought of checking the permission before executing a command?

### ISSUE THREAD

**Issue by yannicklamprecht** [zDevelopers/zLib#26]
https://github.com/zDevelopers/zLib/blob/master/src/main/java/fr/zcraft/zlib/components/commands/CommandGroup.java#L147-L178

ImageOnMap has no Permission check due this bug.

**Comment by AmauryCarrade** [comment]
Yes but we already test that in the `Command.execute()` method—`canExecute` checks the permissions and all. Are you sure there is a bug on the zLib side there?

https://github.com/zDevelopers/zLib/blob/cb43ab335154738947d647b7d3c9e3a84996b180/src/main/java/fr/zcraft/zlib/components/commands/Command.java#L165-L166

**Comment by yannicklamprecht** [comment]
> Yes but we already test that in the `Command.execute()` method—`canExecute` checks the permissions and all. Are you sure there is a bug on the zLib side there?
> 
> https://github.com/zDevelopers/zLib/blob/cb43ab335154738947d647b7d3c9e3a84996b180/src/main/java/fr/zcraft/zlib/components/commands/Command.java#L165-L166

Then it is a bug in here. https://github.com/zDevelopers/ImageOnMap version 4.0 does allow creating new maps without having the needed permissions. So it is clearly a bug.

**Comment by AmauryCarrade** [comment]
Okay. I'll check that and meanwhile, move this issue to the right repository.
